content/blog/introducing-pulumi-import-command/index.md

```diff
@@ -0,0 +1,253 @@
+---
+title: "Introducing The Pulumi Import Command"
+date: "2020-11-23"
+meta_desc: "Introducing the new pulumi import command that will scaffold your Pulumi application code."
+authors: ["paul-stack"]
+tags: ["import","New-Features", "Pulumi-News", "Features"]
+---
+
+In June 2019, Pulumi introduced the ability to import existing infrastructure resources to be part of Pulumi management.
```

Comment:
It would be nice to have something here that explains why we did this. Something like (needs more fleshing out)

"Pulumi users often have existing cloud infrastructure they'd like to import into their infrastructure as code. Importing these resources can often be an arduous task, so we're excited to announce a new richer import experience which makes adding existing cloud resources to your Pulumi program easier than ever"

Reply:
Ok, I re-worded this section!